### DISCUSSION TITLE
Reload instead of HMR

### DISCUSSION THREAD

**Discussion by lttr** [nuxt/nuxt#8929] [Questions (Nuxt 2)]
Is it possible to configure Nuxt to reload current page on file changes?

HMR is not working for me. I start the dev server, edit styles and the first time it shows a the building indicator and when it is done, the style change appears on the screen. The second time it shows the indicator and nothing happens. No more style updates are visible in the browser until I hit F5.

I am using scoped `scss` for styles everywhere.

**Comment by udithishara** [comment]
This could be related to the environment you're working on, (WSL2 specially), could you provide that information?

also try adding this to your `nuxt.config.js`

```
  watchers: {
    webpack: {
      aggregateTimeout: 300,
      poll: 1000,
    },
  },
```

**Reply by lttr** [reply]
I'm on Linux, specifically Pop_OS 20.10.

The additional webpack config does not help, it is the same. First time after manual refresh it works, second code change and my app is not updating any more.
